### PR TITLE
Ignore user ID for display pronoun in messages

### PR DESCRIPTION
**Pull request by User9684**
This PR is useful for ignoring specific users that have abnoxious pronouns, such as a spam or outright slurs.

### CODE REVIEW

**Comment by ImLvna** [comment]
Could you add a context menu entry to user-context to toggle the block without going into settings?

**Comment by ImLvna** [comment]
like copyUserUrls.tsx

File: src/plugins/pronoundb/settings.ts [hunk 1]
```diff
@@ -52,6 +52,11 @@ export const settings = definePluginSettings({
             }
         ]
     },
+    ignoreUsers: {
+        type: OptionType.STRING,
+        description: "Comma-separated list of user IDs to hide pronounds in messages",
```
Comment (lewisakura) [inline src/plugins/pronoundb/settings.ts:57]:
```suggestion
         description: "Comma-separated list of user IDs to hide pronouns in messages",
```

Comment (ImLvna) [reply src/plugins/pronoundb/settings.ts:57]: pronounds😋😋😋😋😋😋

Comment (User9684) [reply src/plugins/pronoundb/settings.ts:57]: oh man I'm so good at typing

**Comment by User9684** [comment]
> Could you add a context menu entry to user-context to toggle the block without going into settings?

Sure!

**Comment by TheKodeToad** [comment]
Personally I'd just block or ban those users in a lot of cases (depending on severity). I thought about opening a PR for an option to just ignore Discord pronouns!

**Comment by User9684** [comment]
Not really an option if the pronoun set is just a bunch of dashes by one of your close friends who just wanna mess with people, or if its in a server you aren't a moderator in

> Personally I'd just block or ban those users in a lot of cases (depending on severity). I thought about opening a PR for an option to just ignore Discord pronouns!

**Comment by TheKodeToad** [comment]
yes - I understand. I feel like there are a *lot* of people abusing this feature though, which is what gave me the other idea. So I'm not necessarily against this idea (and I'm obviously not a maintainer either) - but I can imagine the list getting quite long.

**Comment by User9684** [comment]
> yes - I understand. I feel like there are a _lot_ of people abusing this feature though, which is what gave me the other idea. So I'm not necessarily against this idea (and I'm obviously not a maintainer either) - but I can imagine the list getting quite long.

Haha yeah the list would probably get very long, also, isn't there already an option to block discord pronouns in the plugin?

**Comment by User9684** [comment]
Nevermind, there is not.

**Pull request closed without merging**